mlflow/metrics/genai/metric_definitions.py

```diff
@@ -15,6 +15,7 @@ def correctness(
     model: Optional[str] = None,
     metric_version: Optional[str] = None,
     examples: Optional[List[EvaluationExample]] = None,
+    timeout=60,
```

Comment:
I think we should call this `judge_request_timeout` to be inline with make_genai_metric so we don't deviate with the API and users don't need to learn 2 different arg